src/ext/java/org/opentripplanner/ext/siri/updater/SiriHelper.java

```diff
@@ -104,7 +105,7 @@ private static Siri createETServiceRequest(String requestorRefValue, int preview
     etRequest.setVersion("2.0");
 
     if (previewIntervalMillis > 0) {
-      etRequest.setPreviewInterval(datatypeFactory.newDuration(previewIntervalMillis));
+      etRequest.setPreviewInterval(Duration.ofMillis(previewIntervalMillis));
```

Comment:
Can we remove `datatypeFactory` and `DatatypeFactory` altogether from the class?

Reply:
done